common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -45,6 +48,11 @@
      */
     public static final String ENCODING_UTF8 = "UTF-8";
 
+    /**
+     * The Constant CHARSET_UTF8.
+     */
+    public static final Charset CHARSET_UTF8 = StandardCharsets.UTF_8; //Charset.forName("UTF-8");
```

Comment:
Is our `minSdk` high enough for this?

Reply:
You're right, I mean to leave that out (hence the comment).  If this build passes, we have to figure it out.